### PR TITLE
Updates to Ukraine content following UR

### PR DESCRIPTION
**Pull request by laurahermionetennant92**
Updates to Ukraine content following UR

### CODE REVIEW

**Comment by laurahermionetennant92** [comment]
Hi folks, this is ready for review when you get a sec @EllieNodder @GDW7 
Thanks :)

File: app/views/content/non-uk-teachers/ukraine.md [hunk 1]
```diff
@@ -119,29 +168,37 @@ You can work in an English school and, alongside your job, apply for assessment
...
 * [the Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa)
 * [the Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme)
 * [the Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme)
 
-From 1 August 2022, you will:
+### Deadlines for applications
+
+The closing date for applications to train to teach in 2022/23 is 6 September 2022. Applications for 2023/24 open on 11 October 2022.
+
+If you miss the deadline to apply this year, you could consider finding work as a [teaching assistant](#apply-for-a-teaching-job) to gain valuable experience for your teacher training course.
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/ukraine.md:189]: @laurahermionetennant92 shouldn't really have the same link text that links to different places (bad for SEO and accessibility) - either remove the link on teaching assistant or link to the other teaching assistant page you linked to earlier on?

File: app/views/content/non-uk-teachers/ukraine.md [hunk 2]
```diff
@@ -87,26 +100,66 @@ We understand it might not be possible to provide all this evidence. However, it
...
+
+This service also lists [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) and education support roles. It can be helpful to gain experience of the English education system working in a teaching assistant or support role before you apply for teaching jobs. 
 
 For a short-term unpaid placement, you can [request school experience in a primary or secondary school in England](https://schoolexperience.education.gov.uk/). 
 
 Most teaching vacancies are advertised in March, April or the first half of May, with jobs starting at the beginning of the school year in September. Set up a [Teaching Vacancies](https://teaching-vacancies.service.gov.uk/) job alert to be updated about opportunities throughout the year.
 
+### Other teaching job options
+
+Search online for 'supply teacher' to register with an agency for short-term roles in school. Supply teachers fill in when schools are temporarily short of staff.
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/ukraine.md:142]: Is this a clear enough instruction for users? We can't guarantee which search results they'll see if they type 'supply teacher' so won't know if they're going to the right place. Is there not something more specific we can link them to?

File: app/views/content/non-uk-teachers/ukraine.md [hunk 3]
```diff
@@ -87,26 +100,66 @@ We understand it might not be possible to provide all this evidence. However, it
...
 
 For a short-term unpaid placement, you can [request school experience in a primary or secondary school in England](https://schoolexperience.education.gov.uk/). 
 
 Most teaching vacancies are advertised in March, April or the first half of May, with jobs starting at the beginning of the school year in September. Set up a [Teaching Vacancies](https://teaching-vacancies.service.gov.uk/) job alert to be updated about opportunities throughout the year.
 
+### Other teaching job options
+
+Search online for 'supply teacher' to register with an agency for short-term roles in school. Supply teachers fill in when schools are temporarily short of staff.  
+
+Search your [local authority ('council') website](https://www.gov.uk/find-local-council) for teaching roles.
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/ukraine.md:144]: Does this work? When I tried it, it asked me for my postcode and then when I navigated through to jobs>teaching, it sent me straight back to the GIT website.

Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/ukraine.md:146]: I think maybe it depends on the council website. It worked for me for Lewisham and Manchester.

**Comment by laurahermionetennant92** [comment]
Thanks for looking @EllieNodder :)

File: app/views/content/non-uk-teachers/ukraine.md (hunk 2 above)
Comment (EllieNodder) [reply app/views/content/non-uk-teachers/ukraine.md:144]: @laurahermionetennant92 have you had any more thoughts about this?

**Review by gemmadallmandfe — COMMENTED** [review]
@laurahermionetennant92 @EllieNodder I've added a few more comments

File: app/views/content/non-uk-teachers/ukraine.md (hunk 3 above)
Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/ukraine.md:146]: Ok good point have amended

File: app/views/content/non-uk-teachers/ukraine.md [hunk 4]
```diff
@@ -87,26 +100,68 @@ We understand it might not be possible to provide all this evidence. However, it
...
+* a high standard of written and spoken English – for example, [Common European Framework of Reference for Languages, English level B2](https://www.efset.org/cefr/b2/) 
+
+### Meeting the standard for written and spoken English
+
+Most local authorities (also called 'councils') offer English to Speakers of Other Languages (ESOL) courses as part of adult education.
+
+If you’re in the UK under a Ukrainian visa scheme (Ukraine Sponsorship Scheme, Ukraine Family Scheme or Ukraine Extension Scheme) and aged 19 or over, you are eligible for funding to study an ESOL course (as well as other educational or training courses).
+
+Depending on your circumstances, your ESOL course will either be free or reduced in price. [Visit the council website](https://www.gov.uk/find-local-council) for the area where you live and search ESOL, or, search for courses using the [National Careers Service](https://nationalcareers.service.gov.uk/find-a-course/page?searchTerm=ESOL&distance=10%20miles&town=&orderByValue=Start%20date&startDate=Anytime&courseType=&courseHours=&courseStudyTime=&filterA=true&page=1&D=0&coordinates=&campaignCode=&qualificationLevels=).
```
[the remark this answers is not in the document]
Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/ukraine.md:119]: Have tweaked, see what you think

File: app/views/content/non-uk-teachers/ukraine.md [hunk 5]
```diff
@@ -87,26 +100,68 @@ We understand it might not be possible to provide all this evidence. However, it
...
+
 
 [Find out more about teaching in England if you trained overseas](/non-uk-teachers/teach-in-england-if-you-trained-overseas). 
 
 ### Apply for a teaching job
 
-To find a teaching job, you must apply to individual schools (not DfE). Use the [Teaching Vacancies](https://teaching-vacancies.service.gov.uk/) service to search for jobs by region, subject specialism and age group. This service also lists [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) and education support roles. It can be helpful to gain experience of the English education system working in a teaching assistant or support role before you apply for teaching jobs. 
+To find a teaching job, you must apply to individual schools (not DfE). [Search for teaching jobs](https://teaching-vacancies.service.gov.uk/) by region, subject specialism and age group. 
+
+This service also lists [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) and education support roles. It can be helpful to gain experience of the English education system working in a teaching assistant or support role before you apply for teaching jobs.
```
[the remark this answers is not in the document]
Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/ukraine.md:136]: Yes that's better

File: app/views/content/non-uk-teachers/ukraine.md [hunk 6]
```diff
@@ -119,29 +174,37 @@ You can work in an English school and, alongside your job, apply for assessment
...
+From spring 2023, qualified teachers from Ukraine will be able to apply to the Teaching Regulation Agency for QTS (if they meet the criteria) without paying a fee to a training provider. [Learn more about the way QTS will be awarded to teachers from overseas](https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers).
 
 ### Salaries and benefits
 
 Your salary as a teacher depends on whether you have QTS, and your level of experience. The salary for a teacher without QTS will generally start at £18,419 outside London. [Find out more about salaries and benefits](/salaries-and-benefits) for teachers in England.
 
 ## Train to teach in England
 
-If you are resident in the UK under:
+From 1 August 2022, you will have the right to apply to train to teach in England if you are resident in the UK under:
```
[the remark this answers is not in the document]
Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/ukraine.md:185]: done

File: app/views/content/non-uk-teachers/ukraine.md [hunk 7]
```diff
@@ -119,29 +174,37 @@ You can work in an English school and, alongside your job, apply for assessment
...
 
 * [the Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa)
 * [the Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme)
 * [the Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme)
 
-From 1 August 2022, you will:
+### Deadlines for applications
+
+The closing date for applications to train to teach in 2022/23 is 6 September 2022. Applications for 2023/24 open on 11 October 2022.
```
[the remark this answers is not in the document]
Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/ukraine.md:193]: I will have to do this when I get back on 14th Sept – unless someone else would be able to? :)

File: app/views/content/non-uk-teachers/ukraine.md [hunk 8]
```diff
@@ -191,38 +256,48 @@ The [UK Council for International Student Affairs](https://www.ukcisa.org.uk/) h
...
 
 Given documentation may be difficult for refugees to provide, we expect schools to draw on a varying range of sources on a case by case basis, depending on the individual in question. It is good practice to carry out a risk assessment recording the evidence you have gathered to reach your decision. 
 
 ## Useful links
 
-[Move to the UK if you’re coming from Ukraine](https://www.gov.uk/guidance/move-to-the-uk-if-youre-from-ukraine)
+[Move to the UK if you’re coming from Ukraine](https://www.gov.uk/guidance/move-to-the-uk-if-youre-from-ukraine).
 
-[Welcome guide for Ukrainians in the UK](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1086088/Ukraine_Welcome_Guide.pdf)
+[Welcome guide for Ukrainians in the UK](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1086088/Ukraine_Welcome_Guide.pdf).
```
[the remark this answers is not in the document]
Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/ukraine.md:296]: I don't think so!

File: app/views/content/non-uk-teachers/ukraine.md [hunk 9]
```diff
@@ -158,7 +221,7 @@ You will also need to:
 
 * have the health and physical capacity to train to teach
 * undergo safeguarding and criminal record checks carried out by your training provider or employer
-* show your training provider you meet their standards for maths and English (you’ll usually have to show you’ve passed an English language test, for example, the [International English Language Testing System (IELTS)](https://www.ielts.org/), [Test of English as a Foreign Language (TOEFL)](https://www.ets.org/toefl), or [C2 Proficiency, formerly known as Cambridge English: Proficiency (CPE)](https://www.cambridgeenglish.org/exams-and-tests/proficiency/))
+* show your training provider you meet their standards for maths and English (you’ll usually have to show you’ve passed an [English language test](#meeting-the-standard-for-written-and-spoken-english))
 
 ### Get help applying for teacher training
```
[the remark this answers is not in the document]
Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/ukraine.md:226]: done

File: app/views/content/non-uk-teachers/ukraine.md (hunk 1 above)
Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/ukraine.md]: done

File: app/views/content/non-uk-teachers/ukraine.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (EllieNodder) [reply app/views/content/non-uk-teachers/ukraine.md:193]: @gemmadallmandfe if we hang off merging these changes until Wednesday, we can just take out the first sentence?

If we do this though, the section is talking about deadlines for applications, but we're only talking about when they open, not when they close.

[the remark this answers is not in the document]
Comment (gemmadallmandfe) [reply app/views/content/non-uk-teachers/ukraine.md:193]: @EllieNodder how about this? And then we need a ticket to update this section again in October

```suggestion
Applications to train to teach in 2022/23 have now closed. Applications for 2023/24 open on 11 October 2022.
```

File: app/views/content/non-uk-teachers/ukraine.md [hunk 10]
```diff
@@ -119,29 +174,37 @@ You can work in an English school and, alongside your job, apply for assessment
...
 * [the Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa)
 * [the Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme)
 * [the Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme)
 
-From 1 August 2022, you will:
+### Deadlines for applications
+
+The closing date for applications to train to teach in 2022/23 is 6 September 2022. Applications for 2023/24 open on 11 October 2022.
+
+If you miss the deadline to apply this year, you could consider finding work as a [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) to gain valuable experience for your teacher training course.
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/ukraine.md:195]:
```suggestion
If you missed the deadline to apply this year, you could consider finding work as a [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) to gain valuable experience for your teacher training course. 
```

File: app/views/content/non-uk-teachers/ukraine.md [hunk 11]
```diff
@@ -87,26 +100,68 @@ We understand it might not be possible to provide all this evidence. However, it
...
+
+### Meeting the standard for written and spoken English
+
+Most local authorities (also called 'councils') offer English to Speakers of Other Languages (ESOL) courses as part of adult education.
+
+If you’re in the UK under a Ukrainian visa scheme (Ukraine Sponsorship Scheme, Ukraine Family Scheme or Ukraine Extension Scheme) and aged 19 or over, you are eligible for funding to study an ESOL course (as well as other educational or training courses).
+
+Depending on your circumstances, your ESOL course will either be free or reduced in price. [Find out which local authority ('council')](https://www.gov.uk/find-local-council) you live in, and then search their website for English to Speakers of Other Languages (ESOL) courses. Or, search for courses using the [National Careers Service](https://nationalcareers.service.gov.uk/find-a-course/page?searchTerm=ESOL&distance=10%20miles&town=&orderByValue=Start%20date&startDate=Anytime&courseType=&courseHours=&courseStudyTime=&filterA=true&page=1&D=0&coordinates=&campaignCode=&qualificationLevels=).
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/ukraine.md:119]: @EllieNodder would this work better?

```suggestion
Depending on your circumstances, your ESOL course will either be free or reduced in price. [Find your local authority ('council')](https://www.gov.uk/find-local-council) and then search their website for English to Speakers of Other Languages (ESOL) courses. Or, search for courses using the [National Careers Service](https://nationalcareers.service.gov.uk/find-a-course/page?searchTerm=ESOL&distance=10%20miles&town=&orderByValue=Start%20date&startDate=Anytime&courseType=&courseHours=&courseStudyTime=&filterA=true&page=1&D=0&coordinates=&campaignCode=&qualificationLevels=). 
```

File: app/views/content/non-uk-teachers/ukraine.md [hunk 12]
```diff
@@ -87,26 +100,68 @@ We understand it might not be possible to provide all this evidence. However, it
...
 
 For a short-term unpaid placement, you can [request school experience in a primary or secondary school in England](https://schoolexperience.education.gov.uk/). 
 
 Most teaching vacancies are advertised in March, April or the first half of May, with jobs starting at the beginning of the school year in September. Set up a [Teaching Vacancies](https://teaching-vacancies.service.gov.uk/) job alert to be updated about opportunities throughout the year.
 
+### Other teaching job options
+
+[Supply teachers](https://www.tes.com/jobs/careers-advice/supply-teaching/how-become-supply-teacher) fill in when schools are temporarily short of staff. Register with a supply teacher agency for short-term roles in school.  
+
+[Find out which local authority ('council')](https://www.gov.uk/find-local-council) you live in, and then search their website for teaching roles.
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/ukraine.md:146]: @EllieNodder would this be better

```suggestion
[Find your local authority ('council')](https://www.gov.uk/find-local-council) and then search their website for teaching roles.
```

**Review by gemmadallmandfe — APPROVED** [review]
Happy for this to be merged

**Pull request merged**